plugins/org.wso2.developerstudio.eclipse.esb.dashboard.templates/Samples/HeaderBasedRoutingTemplate/src/main/synapse-config/api/CalculatorAPI.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
same comments for ArithmaticOperationService.xml applies here

Reply:
Fixed.